docs/agent-install-awsecs.md

```diff
@@ -0,0 +1,123 @@
+# Install to AWS EC2
```

Comment:
This doc is describing ECS, **NOT** EC2 directly.  This needs major clarification and rework.

Reply:
I based my work on these links:

- [Configuration Management Tools](https://docs.signalfx.com/en/latest/integrations/agent/advanced-install-options.html#configuration-management-tools), in the section AWS Elastic Container Service (ECS).
- In that section is a link to [ECS Deployment](https://github.com/signalfx/signalfx-agent/tree/master/deployments/ecs). In step 3, I see the note: Select the big "EC2" square and click "Next step". The agent only supports EC2 mode and not Fargate at this time.

I can revise my entry title to **AWS ECS**. What else should change?